### ISSUE TITLE
Bring up CI system

### ISSUE THREAD

**Issue by bluek1te** [bluek1te/feed-txstate#4]
Bring up CI System using Github Actions/Jenkins (TBD)

**Comment by smashwilson** [comment]
On the GitHub Actions front, are you already on the waitlist?

If you haven't yet, visit https://github.com/features/actions and click "Sign up for the beta" to be added. (It'll count as an acknowledgement of our early access terms of service and make the lawyers happy.) Then I may be able to put in a request to expedite your account on behalf of the project.

Failing that, I'd recommend looking into something like [Travis](https://travis-ci.org/dashboard) or [Azure DevOps Pipelines](https://azure.microsoft.com/en-us/services/devops/pipelines/), which both have free tiers for open-source projects.

**Comment by bluek1te** [comment]
Yes, I'm already on the waitlist!

**Comment by bluek1te** [comment]
@smashwilson , would I have to make the repository private in order to be able to using Actions?

**Comment by smashwilson** [comment]
Nope! Once you're in, you can use them on public or private repositories, I believe. (And they'll be free up to some number of build minutes on public ones.)

**Comment by smashwilson** [comment]
And now you're in 😉

**Comment by bluek1te** [comment]
Awesome, thank you so much! Am going to give it a shot soon

**Comment by bluek1te** [comment]
Now compiles on github actions, gotta figure out how to automatically deploy to a webserver

**Comment by smashwilson** [comment]
> automatically deploy to a webserver

Depending on the host solution we choose, we might not need to arrange this ourselves. See #8 for some discussion of the alternatives.
